src/Logger-Arduino_AllPins/Logger-Arduino_AllPins.ino

```diff
@@ -5,8 +5,7 @@ String EOL;
 
 void setup() {
 
-   for (int i=2; i <= 13; i++){
-  
+   for (int i=2; i <= NUM_DIGITAL_PINS; i++){
```

Comment:
When starting with i=2 and ending with i=NUM_DIGITAL_PINS, in total one digital pin will be missed. Is this intended? And also, do all Arduinos start their digital pins on i=2?